torch/_inductor/codegen/wrapper.py

```diff
@@ -951,6 +951,11 @@ def add_fake_input(name, shape, stride, device, dtype):
                 f"device='{device}', dtype={dtype})"
             )
 
+        def add_torchbind_input(name, value):
+            import pickle
+
+            output.writeline(f"{name} = pickle.loads({pickle.dumps(value)!r})")
```

Comment:
are pickle objects round trip serializable via pickle dumps? I thought with torchbind we did serialization via get state and set state.. should we use that ?

Reply:
Yeah I thought the get_state and set_state are used to override the pickle logic?